python/lsst/validate/drp/calcnonsrd/model_phot_rep.py

```diff
@@ -0,0 +1,221 @@
+# LSST Data Management System
+# Copyright 2008-2019 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+import numpy as np
+import astropy.units as u
+
+import lsst.pipe.base as pipeBase
+from lsst.verify import Measurement, Datum
+
+from lsst.validate.drp.calcsrd.pa1 import computeWidths, getRandomDiffRmsInMmags
+
+
+def measureModelPhotRepeat(metric, matches, magKey, filterName, numRandomShuffles=50):
```

Comment:
Why do you need a separate function here?  Isn't all of the logic, particularly in these wrapper functions, essentially exactly the same as for the point-source calculation.

It looks like the documentation is one of the other differences.  I think it would be fine if you just changed the language in the function currently used to do things for stars to generically say "object".